### ISSUE TITLE
Documentation styling in https://cmb2.io/ is totally inaccessile to read

### ISSUE THREAD

**Issue by minamme** [CMB2/CMB2#1481]
**Is your feature request related to a problem? Please describe.**
I want you guys to just adjust documentation [website](https://cmb2.io/) and make it a little more readable by just changing text color to darker variant because currently it's very hard to read on white background and text color `#505050`.
**Describe the solution you'd like**
Just make change like increase text size, color to black for paragraphs and link colors to something like `#125353`

**Comment by tw2113** [comment]
@jtsternberg Definitely agreeing with this one.

@minamme The GitHub Wiki here is also a pretty good resource as well https://github.com/CMB2/CMB2/wiki

**Comment by minamme** [comment]
yes but website documentation is the first choice of many users and even on google

**Comment by angryaxi** [comment]
It seems that https://cmb2.io currently redirects to https://github.com/CMB2/CMB2#cmb2
If the separate documentation site has been permanently retired, this issue can be closed.
